packages/cloud-cognitive/src/components/TagSet/TagSetOverflow.js

```diff
@@ -86,6 +87,16 @@ export const TagSetOverflow = React.forwardRef(
 TagSetOverflow.displayName = componentName;
 
 TagSetOverflow.propTypes = {
+  /**
+   * className
+   */
+  className: PropTypes.string,
+  /**
+   * fixed underneath header
+   */
+  fixed: PropTypes.shape({
+    offset: PropTypes.number,
+  }),
```

Comment:
This doesn't seem to do anything. Was it left over from a different approach? Or is it needed for something I didn't spot?

```suggestion
```